exegol/model/ContainerConfig.py

```diff
@@ -1041,6 +1050,13 @@ def getTextCreationDate(self) -> str:
             return ""
         return datetime.strptime(self.creation_date, "%Y-%m-%dT%H:%M:%SZ").strftime("%d/%m/%Y %H:%M")
 
+    def getComment(self) -> str:
+        """Get the container creation date.
+                If the creation date has not been supplied on the container, return empty string."""
+        if self.comment is None:
+            return ""
+        return self.comment
```

Comment:
I think you can return the `comment` attribute as is with a return type `Optionnal[str]`

Reply:
And an improvement might be made on the `getLabels` generic method, if there is a None returned, no need to add the corresponding label to the container metadata. Currently, the metadata comment will always be added to the docker object even if there is no comment set.